pyomo/contrib/doe/doe.py

```diff
@@ -1575,15 +1607,26 @@ def compute_FIM_full_factorial(
 
             FIM = self._computed_FIM
 
+            det_FIM, trace_FIM, E_vals, E_vecs, D_opt, A_opt, E_opt, ME_opt = (
+                _compute_FIM_metrics(FIM)
+            )
+            """
+            Since we are using the `_compute_FIM_metrics()` to compute the metrics in this chunk of code., This chunk of code is ready to be deleted.
```

Comment:
Please delete if you feel it is no longer needed. It is in the Git history in case we need to find it.